transformations/chinese_simplified_traditional_perturbation/transformation.py

```diff
@@ -0,0 +1,77 @@
+import itertools
+import random
+import opencc
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+
+def chinese_simplified_traditional_perturbation(text,
+                          prob,
+                          seed,
+                          max_outputs,
+                          config
+                          ):
+    random.seed(seed)
+
+    perturbed_texts = []
+
+    if(config == 's2t'):
+        converter = opencc.OpenCC('s2t.json')
+    elif (config == 't2s'):
+        converter = opencc.OpenCC('t2s.json')
+    else:
+        print("Configuration not specified")
+    for _ in itertools.repeat(None, max_outputs):
+        butter_text = ""
+        for chinese_character in text:
+            if random.random() <= prob:
+
+                new_chinese_character = converter.convert(chinese_character)
```

Comment:
I am not familiar with Chinese. Does this work with any character? Could this fail? 
What if there is a character that is not Chinese in the sentence? Like a quote or a company name?

Reply:
If it is in English or another language, then it doesn't do anything with it so it won't fail. :)

Reply:
ok, thanks :)